_includes/ato_grid.html

```diff
@@ -31,8 +31,8 @@ <h2 class="mdl-card__title-text">{{ automation.title }}</h2>
           <!-- Automation Readme url is located in links index 1 -->
           <!-- We will need to design a README.md button -->
           <a href="{{ automation.links[1].url }}"
-            class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect">
-            Read Me
+            class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" style="font-family: Poppins">
```

Comment:
I would imagine that you would want this button to open in a new tab.  That's something to discuss with the team.